data_wrangling/missing.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        missing_cols = [idx for idx in data.isna().sum().to_dict() if int(data.isna().sum().to_dict()[idx]) &gt; 0]
</pre><p>This list comprehension is too complex and could lead to logical errors. Avoid using more than 2 nested function calls (like <code>function_1.function_2.function_3())</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/gbotemi_bolarinwa/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>